File: dotNet/Semantria.Com/Mapping/Configuration/TaxonomyTopic.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using System.Xml.Serialization;

namespace Semantria.Com.Mapping.Configuration
{
    [DataContract(Name = "topic")]
    [XmlRootAttribute("topic", Namespace = "")]
    public class TaxonomyTopic
    {
        [DataMember(Name = "id", EmitDefaultValue = false)]
        [XmlElementAttribute("id")]
        public string Id { get; set; }

        [DataMember(Name = "type")]
        [XmlElementAttribute("type")]
        public string Type { get; set; }
    }
}

File: Java/src/main/java/com/semantria/mapping/configuration/TaxonomyTopic.java
package com.semantria.mapping.configuration;

import javax.xml.bind.annotation.XmlElement;

public class TaxonomyTopic {
    private String id = "";
    private String type = "";

    @XmlElement(name="id")
    public String getId() { return id; }
    @XmlElement(name="type")
    public String getType() { return type; }

    public void setId(String id) { this.id = id; }
    public void setType(String type) { this.type = type; }
}
